runbotics/runbotics-desktop/src/action/microsoft/excel/excel.utils.ts

```diff
@@ -13,3 +13,16 @@ export const hasWorksheetName = (
     && value !== undefined && value !== null
     && 'worksheetName' in value
     && typeof value.worksheetName === 'string';
+
+export const sortNumbersDescending = (array: (number | string)[]): number[] => {
+    const parsedArray = array.map(item => typeof(item) === 'string' ? Number(item) : item);
+    const isArrayNumeric = parsedArray.every(item => typeof(item) === 'number');
```

Comment:
now you're iterating two times over the same array. I'd more effective to make it in one:
let isArrayNumeric = true;
const parsedArray = array.map(item => isNaN(Number(item)) ? isArrayNumeric = false : Number(item))